modules/http/http.c

```diff
@@ -424,11 +452,19 @@ http_dd_init(LogPipe *s)
 
   log_template_options_init(&self->template_options, cfg);
 
+  if (!(self->curl = curl_easy_init()))
+    {
+      msg_error("curl: cannot initialize libcurl", NULL);
+      return FALSE;
```

Comment:
It seems curl_easy_init cannot report back the reason of the error:
https://curl.haxx.se/mail/lib-2009-11/0243.html,
so unfortunalely we cannot give curl-specific hint of the error.

I will insert log_pipe_location_tag() here and into the _insert method when curl_easy_perform is called as you suggest.